lib/com/hydrologis/smash/import_export_plugins/export/gtt/gtt_export.dart

```diff
@@ -84,8 +84,28 @@ class _GttExportWidgetState extends State<GttExportWidget> {
 
   bool _uploadCompleted = false;
   List<Widget> _uploadTiles = [];
-  List<DropdownMenuItem> _projects = [];
+  List<DropdownMenuItem> _projects = [
+    DropdownMenuItem(child: Text(IEL().gttExport_selectProject), value: "none")
+  ];
+  List<DropdownMenuItem<String>> _gpsLogsProj = [
+    DropdownMenuItem(child: Text(IEL().gttExport_selectProject), value: "none")
+  ];
+  List<DropdownMenuItem<String>> _simpleNotesProj = [
+    DropdownMenuItem(child: Text(IEL().gttExport_selectProject), value: "none")
+  ];
+  List<DropdownMenuItem<String>> _imagesProj = [
+    DropdownMenuItem(child: Text(IEL().gttExport_selectProject), value: "none")
+  ];
   late String _selectedProj;
+  String _selectedGpsLogProj = "";
+  String _selectedSimpleNotesProj = "";
+  String _selectedImagesProj = "";
+  String _defaultSubject = "";
+
+  String _simpleTracker = "1000000";
+  String _photoTracker = "1000000";
+  String _gpsTracker = "1000000";
+  String _formDefaultTracker = "1000000";
```

Comment:
@nobnisai @mbasa  
Just curious, but what do these "1000000" default values mean ?

Reply:
Well, I heard that enough large default value is necessary from @mbasa, so let's keep this as it is.